OpenFn/kit
# downgrade tsup

A recent bump to tsup, which we used to build the CLI (and other packages), seems to have caused a bit of a build issue.

It has started bundling package.json into a js chunk and then re-importing it, which causes the version readouts inside the CLI to be wrong.

Kinda hard to explain. I'm downgrading now so that we get a more stable build.

## QA

You can test this without releasing:

On main:

* `pnpm install`
* `pnpm install:openfnx`
* `openfnx test`

The version numbers printed by the test command should be like `workspace:*`,

Now checkout this branch and repeat the same steps (plus clean):
* `pnpm clean:local`
* `pnpm install`
* `pnpm install:openfnx`
* `openfnx test`

The version numbers printed by the CLI should be _local paths_ (this itself is a bit wierd but don't worry about that, it's expected for openfnx!)
